### PR TITLE
Accordion in edit group block

### PR DESCRIPTION
**Pull request by nileshgulia1**

### CODE REVIEW

File: src/components/manage/Blocks/Accordion/Edit.jsx [hunk 1]
```diff
@@ -40,35 +41,45 @@ const Edit = (props) => {
...
-        manage={manage}
-        selectedBlock={selected ? selectedBlock : null}
-        allowedBlocks={data.allowedBlocks}
-        title={data.placeholder}
-        description={data?.instructions?.data}
-        onSelectBlock={(id) => setSelectedBlock(id)}
-        onChangeFormData={(newFormData) => {
+      <AccordionEdit
+        data={data}
+        onChange={(initialData) => {
```
Comment (tiberiuichim) [inline src/components/manage/Blocks/Accordion/Edit.jsx:46]: why is onChange needed here?

**Review by tiberiuichim — CHANGES_REQUESTED** [review]

File: src/components/manage/Blocks/Accordion/Edit.jsx (hunk 1 above)
Comment (nileshgulia1) [reply src/components/manage/Blocks/Accordion/Edit.jsx:46]: Ah, its no need. I'll remove it.

**Comment by nileshgulia1** [comment]
@tiberiuichim Should we also allow editors to add more accordion(s) from the sideBar like in the columns block?

**Review by tiberiuichim — CHANGES_REQUESTED** [review]
What about "data" for the AccordionEdit ?

**Comment by nileshgulia1** [comment]
@tiberiuichim I have made some styling changes along with changed you requested. Please review again.

**Comment by tiberiuichim** [comment]
@nileshgulia1 will it have the content have the same padding in view mode? We try to keep the views as much "WYSIWYG", so that there's not a lot of differences in text wrapping.

**Comment by nileshgulia1** [comment]
@tiberiuichim Yes, I Think content layout in both Edit and View should not have differences. I also added Accordion in View.

**Comment by nileshgulia1** [comment]
@tiberiuichim Would you mind please giving it another shot..?

**Pull request merged**